src/enkf_chgres_recenter_nc.fd/CMakeLists.txt

```
@@ -8,7 +8,10 @@ utils.f90
 )
 
 if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
-  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fp-model precise")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fp-model precise -fpp")
+endif()
+if(ip_VERSION GREATER_EQUAL 4.0.0)
```

```suggestion
if(ip_VERSION VERSION_GREATER 3.9.9)
```
I think we should use the CMake's version compare rather than string compare.
I use 3.9.9 as an estimate for one version less than 4.0.0.
See the if documentation at: https://cmake.org/cmake/help/v3.0/command/if.html